scripts/oneoff/remove-dos4-questions-from-dos5-drafts.py

```diff
@@ -0,0 +1,110 @@
+
+
+def draft_service_contains_dos4_answer(draft: dict) -> bool:
+    invalid_answers = {
+        "agileCoachPriceMin",
+        "businessAnalystPriceMin",
+        "communicationsManagerPriceMin",
+        "contentDesignerPriceMin",
+        "dataArchitectPriceMin",
+        "dataEngineerPriceMin",
+        "dataScientistPriceMin",
+        "deliveryManagerPriceMin",
+        "designerPriceMin",
+        "developerPriceMin",
+        "performanceAnalystPriceMin",
+        "portfolioManagerPriceMin",
+        "productManagerPriceMin",
+        "programmeManagerPriceMin",
+        "qualityAssurancePriceMin",
+        "securityConsultantPriceMin",
+        "serviceManagerPriceMin",
+        "technicalArchitectPriceMin",
+        "userResearcherPriceMin",
+        "webOperationsPriceMin",
+        "developerAccessibleApplications",
+        "contentDesignerAccessibleApplications",
+        "qualityAssuranceAccessibleApplications",
+        "technicalArchitectAccessibleApplications",
+        "webOperationsAccessibleApplications",
+        "serviceManagerAccessibleApplications",
+        "designerAccessibleApplications"
+    }
+
+    return len(invalid_answers.union(set(draft.keys()))) > 0 and draft['status'] == 'not-submitted'
+
+
+def remove_dos4_answers(api_client: DataAPIClient, draft: dict, developer_email: str) -> dict:
+    return api_client.update_draft_service(
+        draft['id'],
+        {
+            "agileCoachPriceMin": None,
+            "businessAnalystPriceMin": None,
+            "communicationsManagerPriceMin": None,
+            "contentDesignerPriceMin": None,
+            "dataArchitectPriceMin": None,
+            "dataEngineerPriceMin": None,
+            "dataScientistPriceMin": None,
+            "deliveryManagerPriceMin": None,
+            "designerPriceMin": None,
+            "developerPriceMin": None,
+            "performanceAnalystPriceMin": None,
+            "portfolioManagerPriceMin": None,
+            "productManagerPriceMin": None,
+            "programmeManagerPriceMin": None,
+            "qualityAssurancePriceMin": None,
+            "securityConsultantPriceMin": None,
+            "serviceManagerPriceMin": None,
+            "technicalArchitectPriceMin": None,
+            "userResearcherPriceMin": None,
+            "webOperationsPriceMin": None,
+            "developerAccessibleApplications": None,
+            "contentDesignerAccessibleApplications": None,
+            "qualityAssuranceAccessibleApplications": None,
+            "technicalArchitectAccessibleApplications": None,
+            "webOperationsAccessibleApplications": None,
+            "serviceManagerAccessibleApplications": None,
+            "designerAccessibleApplications": None
+        },
+        developer_email
+    )
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("stage", type=str)
+    parser.add_argument("developer_email", type=str)
```

Comment:
Can we make this `--updated-by=<developer_email>` to be consistent with other scripts?